lib/services/pages.js

```diff
@@ -27,13 +27,17 @@ const _ = require('lodash'),
  * @param {string} uri
  * @param {{}} data
  * @param {[]} ops
+ * @param {{}} locals
  * @returns {array}
  */
-function addOp(uri, data, ops) {
+function addOp(uri, data, ops, locals = {}) {
+  const { user } = locals;
+
   ops.push({
     type: 'put',
     key: uri,
-    value: JSON.stringify(data)
+    value: JSON.stringify(data),
+    user
```

Comment:
do we need to stringify user too?

Reply:
Good question. Do we need to have the `data` object stringified? Reason I ask is it requires the consumer to do a `JSON.parse` on the data for pretty formatting. In particular, I need to parse it for our datadog logs which means we're doing a lot of extra stringifying/parsing.

Reply:
we do need to parse data, for it's going into postgres as jsonb. user however is not, only key/value is being used from the op to store in the db [here](https://github.com/clay/amphora-storage-postgres/blob/master/postgres/client.js#L141)